Jenkinsfile

```diff
@@ -82,12 +82,14 @@ pipeline {
         stage('Build documentation') {
             steps {
                 withEnv(["WHOME=${env.WORKSPACE}"]) {
-                    sh """
-                        set +x
-                        source /home/saluser/.setup_dev.sh || echo "Loading env failed; continuing..."
-                        setup -r .
-                        package-docs build
-                    """
+                    catchError(buildResult: 'UNSTABLE', stageResult: 'UNSTABLE') {
```

Comment:
Small mod to make it actually work, use

```
catchError(buildResult: 'SUCCESS', stageResult: 'UNSTABLE')
```

here and in line 99 below please.